docusaurus/docs-internals/editor-controls/data-controls/animation.md

```diff
@@ -238,3 +291,130 @@ Enables the ability to change the delay of the animation using a special slider.
   }
 }
 ```
+
+#### Default value example
+
+In the `default` object you can set the default values for any `animation` properties. In this case, the element will
+by default move slightly up on hover.
+
+```js
+{
+  id: "animation",
+  type: "animation",
+  default: {
+    delay: 0,
+    duration: 1000,
+    infiniteAnimation: true,
+    name: "brz-float"
+  }
+}
+```
+
+#### CSS example
+
+This is an abstract example, and usually, you will not manually control any animation. In this case the `map` background 
+color will be red if the animation type is `"pulse"`, otherwise it will be black.
+
+```js
+{
+  id: "animation",
+  type: "animation",
+  style: ({ value }) => {
+    return {
+      "{{WRAPPER}} .brz-ui-ed-iframe": {
+        "background-color": value.type === "pulse" ? "red" : "black"
+      }
+    }
+  }
+}
+```
+
+#### Usage in HTML example 
```

Comment:
in HTML example also need to be implementation of entrance + hover

Reply:
added `entrance` + how to get its values but deleted example how to use its values
I mentioned in `CSS example` that dev doesnt need to control manually any animation